V2.x dev

xhy12 opened the pull request:
修改文档

xhy12 committed:
modified:   docs/user_manual/api_test/intro.md

xhy12 committed:
deleted:    "docs/img/api/API\345\210\227\350\241\250\346\220\234\347\264\242.png"
	deleted:    "docs/img/api/API\345\210\227\350\241\250\351\253\230\347\272\247\346\220\234\347\264\242.png"
	deleted:    "docs/img/api/API\345\210\227\350\241\250\351\253\230\347\272\247\346\220\234\347\264\2421.png"
	deleted:    "docs/img/api/API\346\211\271\351\207\217\346\223\215\344\275\234.png"
	deleted:    docs/img/api/AttachmentArgs.png
	deleted:    "docs/img/api/CASE\345\210\227\350\241\250\345\212\237\350\203\2751.png"

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/metersphere/docs?pullRequest=51) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/metersphere/docs?pullRequest=51) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/metersphere/docs?pullRequest=51) it.</sub>

xhy12 committed:
new file:   "docs/img/api/jp\346\217\220\345\217\202.png"
	new file:   "docs/img/api/xp\346\217\220\345\217\226.png"
	new file:   "docs/img/api/\346\216\250\350\215\220jp.png"
	modified:   docs/user_manual/api_test/api_step.md
